gravity-forms/gw-update-posts.php

```diff
@@ -92,6 +92,32 @@ public function update_post_by_entry( $entry, $form ) {
 			$post->post_name = rgar( $entry, $this->_args['slug'] );
 		}
 
+		if ( $this->_args['date_time'] ) {
+			$date = $this->_args['date_time']['date'];
+			$time = $this->_args['date_time']['time'];
+
+			$date = rgar( $entry, $date );
+			$time = rgar( $entry, $time );
```

Comment:
I would recommend setting the a default time here like so:
 
`rgar( $entry, $time, '00:00 am' );`